src/dmd/cond.d

```diff
@@ -36,13 +36,19 @@ import dmd.func;
 
 /***********************************************************
  */
+
+enum Include
```

Comment:
This gives no information about what this enum is. `Include` is a very generic name.